<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PvPDetailsWidget</class>
 <widget class="QWidget">
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel">
       <property name="minimumWidth">
        <number>48</number>
       </property>
       <property name="text">
        <string>Heros:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="pvpHerosComboBox">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="discardHeroButton">
       <property name="text">
        <string>Discard Hero</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addHeroButton">
       <property name="text">
        <string>Add New Hero</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
     <layout class="QVBoxLayout">
      <item>
       <widget class="HeroDetailsWidget" name="pvpHeroDetails"/>
      </item>
     </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
